pypoptools/pypoptesting/framework/entities.py

```diff
@@ -153,14 +152,14 @@ class VbkBlockResponse:
     blockhash: Hexstr
     blockheight: int
     vbkblock: VbkBlock
-    containingBlocks: List[Hexstr]
+    containingBlocks: list[Hexstr]
 
 
 @dataclass
 class RawPopMempoolResponse:
-    vbkblocks: List[Hexstr]
-    vtbs: List[Hexstr]
-    atvs: List[Hexstr]
+    vbkblocks: list[Hexstr]
```

Comment:
revert back to `List[Hexstr]`. Because this is valid only in python3.9, and `List[Hexstr]` is valid in earlier versions

Reply:
https://stackoverflow.com/a/39458225/1953079

Reply:
fixed